.github/workflows/build.yml

```diff
@@ -9,10 +9,10 @@ on:
 
 jobs:
     build:
-        runs-on: ubuntu-latest
+        runs-on: ubuntu-22.04
```

Comment:
Follow up issue should be created to remind us to revert this once we exit Jazzband.